_data/simulation_list.yaml

```diff
@@ -73,15 +73,15 @@
 - simulations/fenics_1c_ivan/meta.yaml
 - simulations/prismspf_6b/meta.yaml
 - simulations/inl_moose_6a/meta.yaml
-- simulations/moose_2b_jah/meta.yaml
-- simulations/moose_2a_jah/meta.yaml
-- simulations/moose_2d_jah/meta.yaml
-- simulations/moose_1a_jah/meta.yaml
-- simulations/moose_1b_jah/meta.yaml
-- simulations/moose_2c_jah/meta.yaml
+# - simulations/moose_2b_jah/meta.yaml
```

Comment:
Why are these excluded? Is it more appropriate to comment them out, or cut the lines?

Reply:
The data for those results is stored in [jah5759](https://github.com/jah5759)'s GitHub account. The account seems to have disappeared. Another good reason to switch to a DOI service.

I think it's best to comment as it kind of remind me to investigate it later or at least that those results have been skipped.

Reply:
@jah5759

Reply:
Now purged